app/assets/javascripts/discourse/app/components/create-account.js

```diff
@@ -21,6 +21,23 @@ export default Component.extend({
     }
   },
 
+  actionOnEnter(event) {
```

Comment:
@janzenisaac you still need to `@bind` this method, with out it `this.action()` throws an error (and `this.disabled` is always falsy)

I'd bind the other method too, it's a good code hygiene to bind all even listeners, even if they don't currently use `this`. 🙂

Reply:
Will update :)

Reply:
https://github.com/discourse/discourse/pull/17100